grr/gui/webauth.py

```diff
@@ -88,6 +88,29 @@ def SecurityCheck(self, func, request, *args, **kwargs):
     return response
 
 
+class HTTPWebAuthManager(BaseWebAuthManager):
+  """Manager using external, HTTP based authention (e.g. 
+  shibboleth on apache as a reverse proxy in front of GRR)
+  This Manager assumes that access through the proxy is 
+  well controlled.
+  """
+
+  def SecurityCheck(self, func, request, *args, **kwargs):
+    """Wrapping function."""
+   
+    request.user = request.META.get("HTTP_REMOTE_USER")
```

Comment:
I have doubts about this line: request is supposed to be a Werkzeug request object (see https://github.com/google/grr/blob/master/grr/gui/wsgiapp.py#L121) and consequently it shouldn't have the "META" attribute. "META" is part of Django's request objects, but we removed Django request a few months ago. Have you tested this patch with GRR HEAD?